### ISSUE TITLE
Unable to import files having .pcss extension (only windows)

### ISSUE THREAD

**Issue by Nabeelhassan** [jonathantneal/postcss-partial-import#36]
v3.1.0 is still not working on windows (10) x64
The fix that you did for https://github.com/jonathantneal/postcss-partial-import/issues/30 is only working for Linux and Mac.

My config is as follows
```
require("postcss-partial-import")({
	extension: ".pcss",
	prefix: '_'
})
```

old way
```
@import "inc/variables";
```

but this is what I have to do to import the file 
```
@import "inc/_variables.pcss";
```

**Comment by asifshirazi** [comment]
Please accept the PR https://github.com/jonathantneal/postcss-partial-import/pull/34, thanks @frankwallis

**Comment by jonathantneal** [comment]
Thanks for reporting this. It has been resolved by @frankwallis and published in 3.1.1.
